### DISCUSSION TITLE
Are code actions supported?

### DISCUSSION THREAD

**Discussion by Grafcube** [castwide/solargraph#561] [Q&A] [answered]
I recently started learning ruby and I'm using neovim with nvim-lspconfig. When I try to trigger code actions I get this message:
```
method textDocument/codeAction is not supported by any of the servers registered for the current buffer
```
I don't know much about the specifics on how language servers work but looking at this file am I correct in assuming this is not available yet? Or is this something else entirely and it's just my configuration that's wrong? I can post my config here if that's the case.

[./lib/solargraph/language_server/message/text_document/code_action.rb](https://github.com/castwide/solargraph/blob/a550eba4aafb7997ac92ccb5c3da356218250e41/lib/solargraph/language_server/message/text_document/code_action.rb)
```ruby
def process
  # @todo Find code actions
end
```
Thanks for any help.

**Comment by castwide** [comment, marked as answer]
Code actions are not supported yet. They're on the roadmap, but I'm afraid I don't have an ETA.
